src/dreammaker/lexer.rs

```diff
@@ -291,6 +294,7 @@ impl Token {
                 QuestionMark |
                 BitXorAssign |
                 BitOrAssign |
+                SafeBitOrAssign |
```

Comment:
Missing `&&=` and `:=` here, although this function is just for debugging/pretty-printing

Reply:
Yeah, I got confused about the ordering in the middle of adding this...